geode-core/src/integrationTest/java/org/apache/geode/internal/cache/PartitionedRegionStatsJUnitTest.java

```diff
@@ -477,4 +480,27 @@ private long getMemBytes(PartitionedRegion pr) {
 
     return bytes;
   }
+
```

Comment:
This section may need to be struck as  and stuff related to clear came from GEODE-7665

Reply:
Done